src/components/Photo/Selfie.js

```diff
@@ -84,13 +91,9 @@ class SelfieCapture extends Component<Props, State> {
   onUserMedia = () => {
     if (this.props.useMultipleSelfieCapture) {
       // A timeout is required for this.webcam to load, else 'webcam is null' console error is displayed
-      // despite an actual camera stream snapshot being captured
-      // 750ms is the minimum possible timeout without resulting in a null blob being sent to
-      // the /snapshots endpoint in file payload on some browsers, e.g. macOS Firefox & Safari
-      const initialSnapshotTimeout = 750
+      const initialSnapshotTimeout = 0
```

Comment:
Does this also now resolve the issue where macOS Firefox, Safari was sending a `null` blob?

Reply:
Yes, the reason for this PR is that the same issue was now happening on all browsers regardless of timeout

Reply:
Ooof. But that's also good that this resolves it for all now.